benefits/static/css/admin/styles.css

```diff
@@ -0,0 +1,124 @@
+:root {
+  --primary-color: #045b86;
+  --primary-color-rgb: 4, 91, 134;
+  --bs-blue: var(--primary-color);
+  --bs-primary: var(--primary-color);
+  --bs-primary-rgb: var(--primary-color-rgb);
+  --text-primary-color: #212121;
+  --bs-body-color: var(--text-primary-color);
+  --bs-body-text-align: left;
+  --bs-body-font-weight: 400;
+  --medium-font-weight: 500;
+  --bold-font-weight: 700;
+  --bs-body-font-size: calc(18rem / 16);
+  --bs-font-sans-serif: Roboto, system-ui, -apple-system, "Segoe UI", sans-serif,
+    "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", "Noto Color Emoji";
+  --bs-body-font-family: var(--bs-font-sans-serif);
+  --bs-body-line-height: 1.5; /* Displayed as 150% on Figma */
+  --heading-line-height: 1.3; /* Displayed as 130% on Figma */
+  --form-field-text-line-height: 1.25; /*Displayed as 125% on Figma */
+  --h4-line-height: 1.2; /* Displayed as 120% on Figma */
+  --letter-spacing-2: 0.02; /* Displayed as 2% on Figma */
+  --letter-spacing-3: 0.03; /* Displayed as 3% on Figma */
+  --letter-spacing-5: 0.05; /* Displayed as 5% on Figma */
+  --dark-color: #212121;
+  --hover-color: #044869;
+  --error-color: #ea1010;
+  --selected-color: #562b97;
+  --focus-color: #fdb714;
+  --secondary-bg-color: #fafafa;
+  --error-color-rgb: 234, 16, 16;
+  --bs-danger: var(--error-color);
+  --bs-danger-rgb: var(--error-color-rgb);
+  --standout-color: #323a45;
+  --h1-text-align: left;
+  --h3-font-size: calc(20rem / 16);
+  --font-size-24px: calc(24rem / 16);
+  --font-size-20px: calc(20rem / 16);
+  --font-size-18px: calc(18rem / 16);
+  --font-size-15px: calc(15rem / 16);
+  --font-size-16px: calc(16rem / 16);
+  --font-size-14px: calc(14rem / 16);
+  --font-size-12px: calc(12rem / 16);
+  --border-width: calc(2rem / 16);
+  --border-radius: calc(3rem / 16);
+  --focus-style: 3px solid var(--focus-color);
+  --focus-border-radius: calc(2rem / 16);
+  --focus-outline-offset: calc(2rem / 16);
+  --focus-border-radius: calc(2rem / 16);
```

Comment:
RE: https://github.com/cal-itp/benefits/pull/2313/files#r1731506620

It feels like a lot of these variables could probably be moved to a separate file for re-use.

Reply:
Yeah we could in the future, but I'd only be comfortable doing that if Design is 10000% committed to these design tokens.

Reply:
> I'd only be comfortable doing that if Design is 10000% committed to these design tokens.

If Design changes their minds about some of these variables, wouldn't it be easier to handle that update if we had a single `variables.css` file?

Reply:
> Yeah we could in the future, but I'd only be comfortable doing that if Design is 10000% committed to these design tokens.

Does this matter? Are we really coordinated with design on these tokens right now?

Reply:
Yeah but I guess what I mean by "committing" is that if they change one thing in one place, they have to commit to changing it everywhere. Like, let's say, we bump up the H1 size on one page, or focus ring style on one page, we have to commit to those tokens being changed on all pages. Or else we have to add new variables.

Reply:
Created a `variables.css` file in `static/css` and imported it into `css/admin/styles.css` in c7621550bd559933dfa129da1a924a63bc9589b0. Made a minor refactor to import `variables.css` into the benefits app's `css/styles.css` in a88cc400a2e4be1b0ec9e68723d0ea4c10a986db